lib/resource_dsl/acts_as_async_update.rb

```diff
@@ -5,18 +5,26 @@ module ResourceDSL
   # config.batch_actions = true
   # config.scoped_collection_actions_if = -> { true }
   module ActsAsAsyncUpdate
-    def acts_as_async_update(model_class, attrs_to_update)
+    def acts_as_async_update(form_class)
       scoped_collection_action :async_update,
                                title: 'Update batch',
                                class: 'scoped_collection_action_button ui',
-                               form: attrs_to_update,
+                               form: form_class.form_data,
```

Comment:
```suggestion
                               form: -> { form_class.form_data },
```

Reply:
following README of https://github.com/activeadmin-plugins/active_admin_scoped_collection_actions form param should be lambda

Reply:
+